cumulus/client/consensus/aura/src/collators/lookahead.rs

```diff
@@ -260,8 +262,7 @@ where
 				relay_parent,
 				params.para_id,
 				&mut params.relay_client,
-				// Use depth 0, to preserve behaviour.
-				ClaimQueueOffset(0),
+				ClaimQueueOffset(DEFAULT_CLAIM_QUEUE_OFFSET),
```

Comment:
Didn't we want to use 0 to preserve old behaviour ?

Reply:
the problem with using 0 is this:

we have a `DEFAULT_CLAIM_QUEUE_OFFSET` of 1 (as you also suggested).

The default implementation of the `SelectCore` trait uses DEFAULT_CLAIM_QUEUE_OFFSET.

if we instead use here offset 0 and the runtime is updated to use offset 1, there will be a discrepancy and the collator will not advertise to the right core. we need these two places to use the same offset.

Reply:
Explanation makes sense to me, offset of 1 should also not break anything here.

Reply:
an alternative would be to set the `DEFAULT_CLAIM_QUEUE_OFFSET` to 0 and have some kind of `LookaheadCoreSelector` impl of `SelectCore` that uses a claim queue offset of 1.

the current state of the PR (with lookahead using offset 1) breaks the coretime-shared-core zombienet test because core sharing doesn't work between paras if one of the paras is not producing blocks. will be fixed by https://github.com/paritytech/polkadot-sdk/pull/5461

Reply:
actually I think using a default claim queue offset of 0 is a must, because if the UMP signal is not present, the runtime will assume offset 0: https://github.com/paritytech/polkadot-sdk/pull/5423

Reply:
and another reason to use default offset 0 is because the claim queue size is at least 1 (so an offset of 1 will not work on a network that is configured with size 1)

Reply:
> an alternative would be to set the DEFAULT_CLAIM_QUEUE_OFFSET to 0 and have some kind of LookaheadCoreSelector impl of SelectCore that uses a claim queue offset of 1.

I've implemented this now